### ISSUE TITLE
Ignore node_modules/vscode/typings/node.d.ts and use one of my own

### ISSUE THREAD

**Issue by rafaelmaiolla** [Microsoft/vscode#4397]
- VSCode Version: 0.11.8
- OS Version: Window 7

I'm developing an extension, and because the node.d.ts from vscode is not complete. I'm getting the following errors when running `node ./node_modules/vscode/bin/compile`

```bash
src/lib/RemoteFile.ts(103,7): error TS2346: Supplied parameters do not match any signature of call target.
src/lib/Session.ts(75,58): error TS2339: Property 'indexOf' does not exist on type 'Buffer'.
```
The code in these lines are the following;
```javascript
fs.writeSync(this.fd, buffer, offset, length);
```
and
```javascript
buffer.indexOf(line);
```
which are valid, but as the node.d.ts is not complete, they are being considered invalid.

How can I ignore `node_modules/vscode/typings/node.d.ts` and use one of my own?

**Comment by bpasero** [comment]
@rafaelmaiolla lets try to make the d.ts file complete. If you open a PR on https://github.com/Microsoft/vscode-extension-vscode I can merge it in.

**Comment by bpasero** [comment]
@rafaelmaiolla I noticed that the node.d.ts file used in vscode module is a bit outdated, I will update it next week with the latest for 4.x and see that your compile errors are also fixed. Until then, please use the `<any>` cast trick to disable TypeScript checking in this case.

**Comment by bpasero** [comment]
@rafaelmaiolla I published vscode@0.11.9 with an updated node.d.ts
